src/_pytest/fixtures.py

```diff
@@ -1031,48 +1031,6 @@ def __call__(self, function):
         return function
 
 
-FIXTURE_ARGS_ORDER = ("scope", "params", "autouse", "ids", "name")
-
-
-def _parse_fixture_args(callable_or_scope, *args, **kwargs):
```

Comment:
TBH I'm OK with this implementation; I like that it is general enough that we can reuse it eventually when needed.

Having said that if others prefer the new version I'm OK with that too.

Reply:
I don't think we would ever want to reuse it, because it's deprecated and new code should be keyword-only from the start. Any new parameters to `pytest.fixture()` would be keyword-only too. So this whole code block is "read-only" and I think it's better to have it straightforward and localized (and shorter).

Reply:
Yeah I have that feeling like I wanted to reuse this somewhere, but can't really remember it right now. 😁